models/gpt2/model.py

```diff
@@ -0,0 +1,196 @@
+from os import path
+import functools
+
+import h5py
+import jax
+import jax.numpy as jnp
+from jax._src import abstract_arrays
+import numpy as np
+
+
+# pylint: disable=invalid-name
+def load_gpt2_model(name, gpt2_dir):
+  """Load GPT-2 pretrained weights into JAX DeviceArrays."""
+  # TODO(jekbradbury): add target dtype
```

Comment:
🤔 change the TODO name or update attribution for this file?

Reply:
Removed, was carry over from the originally demo code.